lib/lbt/bundle/Builder.js

```diff
@@ -542,15 +547,43 @@ class BundleBuilder {
 		});
 	}
 
-	writeBundleInfos(sections) {
+	logStringBundlingError(moduleName) {
+		log.error(
+			"Module " + moduleName + " requires top level scope and can only be embedded as a string " +
+			"(requires 'eval'), which is not supported with specVersion 4.0 and higher. " +
+			"For more information, see the UI5 Tooling documentation  " +
```

Comment:
Is there a purpose for the two spaces in between the text and the link?

Reply:
good catch :-D